source/Octostache/Templates/Functions/TextValidationFunction.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Octostache.Templates.Functions
+{
+    internal static class TextValidationFunction
```

Comment:
Would `TextComparisonFunctions` be better?

Reply:
Was considering that after I made the name. I agree it makes more sense than validation